documentation/Demo-Server-Guide.md

```diff
@@ -127,34 +124,54 @@ Logs are in `/var/log/nginx`
 docker-compose will complain if directories are not present, create them if needed (they will persist when restarting image, even if `docker-compose down` was run)
 
 ```bash
-mkdir app_snapshots_on_port_8000
-mkdir app_postgres_on_port_8000
+mkdir conforma_on_port_8000 \
+  conforma_on_port_8000/postgres \
+  conforma_on_port_8000/snapshots \
+  conforma_on_port_8000/files \
+  conforma_on_port_8000/localisation \
+  conforma_on_port_8000/preferences
```

Comment:
We'll need to re-do this on the demo server once. But do it on one instance and make sure it is stable before changing the others.